### ISSUE TITLE
Training Log?

### ISSUE THREAD

**Issue by JialianW** [hustvl/QueryInst#12]
Dear authors,

Do you have training log files available? BTW, I got 3 warnings as shown below when training. I was wondering if you also got them? Is that fine?

1. queryinst/mmdet/models/backbones/resnet.py:400: UserWarning: DeprecationWarning: pretrained is a deprecated, please use "init_cfg" instead warnings.warn('DeprecationWarning: pretrained is a deprecated

2. python3.7/site-packages/mmcv/cnn/bricks/conv_module.py:107: UserWarning: ConvModule has norm and bias at the same time
  warnings.warn('ConvModule has norm and bias at the same time') 

3. [W reducer.cpp:346] Warning: Grad strides do not match bucket view strides. This may indicate grad was not created according to the gradient layout contract, or that the param's strides changed since DDP was constructed.  This is
 not an error, but may impair performance. grad.sizes() = [80, 256, 1, 1], strides() = [256, 1, 256, 256]

**Comment by Yuxin-CV** [comment]
Hi @JialianW, thanks for your interest in our work!

As for the logs, we will release them in the near future and we will let you know in the first place, please stay tuned :)

As for the warnings, it is fine and just let the bullets fly.

**Comment by JialianW** [comment]
Thanks. For the learning rate, do we need to change two times smaller when using 4 GPUs? The default learning rate is for 8 GPUs?

**Comment by vealocia** [comment]
@JialianW For learning rate adjustment, please follow the ```Linear Scaling Rule```: smaller batch size with smaller learning rate, vice versa.

**Comment by Yuxin-CV** [comment]
> @JialianW For learning rate adjustment, please follow the `Linear Scaling Rule`: smaller batch size with smaller learning rate, vice versa.

Hi @JialianW.

To be precise, if you use `0.5x` batchsize, then the `linear scaling rule` (https://arxiv.org/abs/1706.02677) recommends using `0.5x` lr optimized via `SGD`. But `QueryInst` & `Sparse R-CNN` as well as `DETR` families use `Adam` families to optimize, therefore the `linear scaling rule` might be too harsh. _E.g._, DETR does not scale their lr even with `batchsize = 64`. You can refer to some issues in the DETR repo (https://github.com/facebookresearch/detr/issues/48#issuecomment-638689380) for details.

For `QueryInst`, we do use the `linear scaling rule` when scaling up the batchsize. But I am not sure if there exists a better principle for `QueryInst` & `Adam` families.

**Comment by Yuxin-CV** [comment]
This issue won't be closed until the training logs are released.
